src/main/java/org/generationcp/middleware/manager/StudyDataManagerImpl.java

```diff
@@ -1318,4 +1344,27 @@ void setDataSetBuilder(final DataSetBuilder dataSetBuilder) {
 	void setTrialEnvironmentBuilder(final TrialEnvironmentBuilder trialEnvironmentBuilder) {
 		this.trialEnvironmentBuilder = trialEnvironmentBuilder;
 	}
+
+	private boolean hasInvalidValues(List<FieldMapLabel> labels) {
```

Comment:
Rename to `hasInvalidCoordinateValue`

Reply:
can be simplied to one line:
return labels.stream().anyMatch(fieldMapLabel -> Util.getIntValue(fieldMapLabel.getColumn()) <= 0 || Util.getIntValue(fieldMapLabel.getRange()) <= 0);